pkgs/by-name/ne/neovim-unwrapped/package.nix

```diff
@@ -113,6 +114,11 @@ stdenv.mkDerivation (
       ./system_rplugin_manifest.patch
     ];
 
+    patchPhase = ''
+      # loosen the version requirement for wasmtime to use the nixpkgs version
```

Comment:
Please, move the comment out of the nix string. Like so, we will be able to modify it without causing any rebuilds.